### PR TITLE
BP fix python version issue

### PR DESCRIPTION
**Pull request by laq0110**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 with open(os.path.join(os.path.dirname(__file__), "gcd/VERSION"), 'r') as vf:
     my_version = vf.read().strip()
 
+with open(os.path.join(os.path.dirname(__file__), "gcd/VERSION"), 'r') as vf:
+    my_version = vf.read().strip()
+
+
```
Comment (memeplex) [inline setup.py]: This is duplicating the previous lines.

File: gcd/VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.1.0
+1.1.2
```
Comment (danielfujii-jampp) [inline gcd/VERSION]: why not 1.1.1?

Comment (laq0110) [reply gcd/VERSION]: In PyPI is not allowed re-upload a package. The 1.1.1 was used as a test.

File: setup.py (hunk 1 above)
Comment (memeplex) [reply setup.py]: Please rename to `version` and you have a LGTM.

**Comment by memeplex** [comment]
We should consider adding a flake8 pre-commit hook for all the projects. Will ask @tzulberti-jampp about this.

**Pull request merged**